docs/getting_started/installation.rst

```diff
@@ -136,6 +136,21 @@ install the assets:
     bin/console cache:clear
     bin/console assets:install
 
+Webpack Encore Integration
+--------------------------
+
+When the assets are installed add following to ``config/sonata_admin.yaml``
+
+.. code-block:: yaml
+
+    webpack_encore:
+        builds:
+            sonata_admin: '%kernel.project_dir%/public/bundles/sonataadmin/dist'
+            # Handling multiple entry points. For example app entry
+            #app: '%kernel.project_dir%/public/build'
+
+This will load the assets to the template.
```

Comment:
This sentence sounds a bit weird. Shall we reword or completely remove it?

Reply:
We can remove it.